### PR TITLE
Fix ATS not finding python

### LINKED ISSUE #943: Add new ats-config for Toss4
forgot to add a new json for toss4 to `ats-config` directory. im not sure if/ how this is affecting ats tests on toss4 but its an easy fix

### PR DESCRIPTION
**Pull request by chapman39**
ATS does not work currently due to me forgetting to add python to the list spack views. If you try to run `ats.sh`, it will try and fail to look for a python within the view directory. This PR will fix this issue. (I'll try to test devtools a little more thoroughly from now on..)

Devtools are rebuilt.

This PR also fixes #943

### CODE REVIEW

File: scripts/spack/devtools_configs/toss_4_x86_64_ib/spack.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ spack:
   view:
     default:
       root: ../view
-      select: [llvm, cppcheck, doxygen, py-ats, py-sphinx]
+      select: [llvm, cppcheck, doxygen, py-ats, py-sphinx, python]
```
Comment (white238) [inline scripts/spack/devtools_configs/toss_4_x86_64_ib/spack.yaml]: I found i also needed this when I ran into it on Axom
```suggestion
      select: [llvm, cppcheck, doxygen, py-ats, py-sphinx, python, ^python]
```

**Comment by chapman39** [comment]
devtools are built, not sure how to re-run an azure test but seems like it was a http timeout

https://dev.azure.com/llnl-serac/serac/_build/results?buildId=9358&view=logs&j=98581895-26e3-5fa9-086f-aab4f8eff071&t=fd9c7d4e-7c37-5e8a-7a17-67235f2444c5&l=13

**Comment by white238** [comment]
> devtools are built, not sure how to re-run an azure test but seems like it was a http timeout
> 
> https://dev.azure.com/llnl-serac/serac/_build/results?buildId=9358&view=logs&j=98581895-26e3-5fa9-086f-aab4f8eff071&t=fd9c7d4e-7c37-5e8a-7a17-67235f2444c5&l=13

I retriggered it but you can always push a noop change to rerun the CI if you don't want to wait for me.

**Comment by white238** [comment]
This should be good to go once you update the paths in the host-configs with the new devtools. Thanks for taking care of this @chapman39

**Review by white238 — APPROVED** [review]

**Review by jamiebramwell — APPROVED** [review]

**Pull request merged**